test/e2e/instrumentation/logging/stackdrvier/basic.go

```diff
@@ -107,6 +108,35 @@ var _ = instrumentation.SIGDescribe("Cluster level logging implemented by Stackd
 				err = utils.WaitForLogs(c, ingestionInterval, ingestionTimeout)
 				framework.ExpectNoError(err)
 			})
+
+			ginkgo.By("Checking that too long lines are trimmed", func() {
+				originalLength := 200 * 1024
+				trimSuffix := "[trimmed]"
+
+				longLine := strings.Repeat("a", originalLength)
+				pod, err := utils.StartAndReturnSelf(utils.NewRepeatingLoggingPod("synthlogger-4", longLine), f)
+				framework.ExpectNoError(err, "Failed to start a pod")
+
+				ginkgo.By("Waiting for logs to ingest")
+				c := utils.NewLogChecker(p, func(_ string, logEntries []utils.LogEntry) (bool, error) {
+					if len(logEntries) == 0 {
+						return false, nil
+					}
+					log := logEntries[0]
+					if log.JSONPayload != nil {
+						return false, fmt.Errorf("got json log entry %v, wanted plain text", log.JSONPayload)
+					}
+					if len(log.TextPayload) == originalLength {
+						return false, fmt.Errorf("got non-trimmed entry of length %d", len(log.TextPayload))
+					}
+					if !strings.HasSuffix(log.TextPayload, trimSuffix) {
```

Comment:
Umm, prefix?

Reply:
I shrank the added part and moved it to the end of the message, because then it will clearly signalize that this is not an end of the original message